.gitignore

```diff
@@ -13,6 +13,7 @@ db.sqlite3
 
 
 # Distribution / packaging
+.idea
```

Comment:
I think this is an example of something small that you can do in different PR and push it to merge easily

Reply:
Done.
A new PR has been created.

Reply:
Done